content/api/entity-reference/opportunity/index.html

```diff
@@ -11,146 +11,16 @@
 
 <h1>{{ name}}</h1>
 
-<p>Updated: November 30, 2016</p>
+<p>Updated: March 7, 2017</p>
```

Comment:
Nah man, today is March 7th and I updated it today ;)

it also doesn't matter much since opportunity doesn't get lit up yet